docs/use-cases/tokenization/index.page.tsx

```diff
@@ -0,0 +1,389 @@
+import React, { useRef, useState } from "react";
+import { useTranslate } from "@portal/hooks";
+import { NavList } from "shared/components/nav-list";
+
+const { translate } = useTranslate();
+
+export const frontmatter = {
+  seo: {
+    title: "XRPL Overview",
+    description:
+      "An introduction to the key benefits and features of the XRP Ledger, a public blockchain driven by the XRPL developer community.",
+  },
+};
+
+const useCases = [
+  {
+    description: "Stablecoin Issuer",
+    link: "/docs/use-cases/tokenization/stablecoin-issuer/",
+  },
+  {
+    description: "NFT Marketplace",
+    link: "/docs/use-cases/tokenization/nftoken-marketplace/",
+  },
+  {
+    description: "Authorized Minter",
+    link: "/docs/use-cases/tokenization/authorized-minter/",
+  },
+  {
+    description: "Digital Artist",
+    link: "/docs/use-cases/tokenization/digital-artist/",
+  },
+];
+
+const securityAdvantages = [
+  {
+    title: "Trustlines & Authorized Trustlines",
+    description: (
+      <>
+        <div className="bold">{translate("Trustlines:")}</div>
+        <div className="card-text">
+          {translate("No spamming of wallets without permission.")}
+        </div>
+        <div className="bold">{translate("Authorized Trustlines:")}</div>
+        <div className="card-text">
+          {translate(
+            "Enable specific accounts to hold issued tokens by setting up allowlisting."
+          )}
+        </div>
+      </>
+    ),
+  },
+  {
+    title: "Freeze & Clawbacks",
```

Comment:
The two relevant links for this category are:

`/docs/concepts/tokens/fungible-tokens/freezes/`
`/docs/concepts/tokens/fungible-tokens/clawing-back-tokens/`

Not sure how you would link to both though.